dev-packages/cli/src/theia.ts

```diff
@@ -156,6 +156,12 @@ function rebuildCommand(command: string, target: ApplicationProps.Target): yargs
                     describe: 'Controls whether to pack or unpack plugins',
                     boolean: true,
                     default: false,
+                },
+                'tolerate_download_failures': {
```

Comment:
The name of the flag can be improved:

```suggestion
                'failOnError': {
```

Reply:
`--ignore-errors` is more concise indeed.